src/FCM/Message/ApnsConfig.php

```diff
@@ -1,8 +1,45 @@
 <?php
-namespace Plokko\Firebase\FCM\Message;
 
+namespace Plokko\Firebase\FCM\Message;
 
+/**
+ * Class ApnsConfig
+ * @package Plokko\Firebase\FCM\Message
+ * @see https://firebase.google.com/docs/reference/fcm/rest/v1/projects.messages#apnsconfig
+ 
+ * @property array $headers
+ * @property ApnsNotification $payload
+ */
 class ApnsConfig
```

Comment:
Shouldn't it implement implement JsonSerializable?!?

Reply:
Yes, it should. I'll fix it too)

Reply:
Fixed this!